.circleci/config.yml

```diff
@@ -20,6 +20,13 @@ aliases:
     name: Install dependencies
     command: yarn install
 
+  - &run_yarn_lint
+    name: Lint & Code styles
+    command: |
+      (cd examples/vue3-ssr-vite && yarn install)
+      (cd examples/vue3-ssr-vue-cli && yarn install)
+      yarn lint
```

Comment:
https://app.circleci.com/pipelines/github/algolia/vue-instantsearch/1295/workflows/e8bc70c9-9e12-4b96-bc1a-34e2d830fa27/jobs/3774

CI failed because there are some packages in the examples that aren't available at the root level. So we have to install the deps before running eslint. Not sure if there's a better alternative.

Reply:
- workspaces
- installing those at root too
- ignoring lint in the examples

Reply:
* workspaces: It could take very long and will make it hard to maintain both master and here, so I'd like to do this after merge of `feat/vue3-compat` (if we decide to do)
* installing those at root too: they're not used elsewhere and it will be confusing to us why we have them like two months later
* ignoring lint in the examples: seems realistic, also considering the fact that we're getting prettier and eslint plugin working on vs code.

the current version (installing the deps in CI) vs ignoring lint in the examples

I prefer the latter. What do you think?

Reply:
I'm ok with ignoring the examples for now